hadoop-tools/hadoop-aws/src/site/markdown/tools/hadoop-aws/performance.md

```diff
@@ -276,7 +283,45 @@ Fix: Use one of the dedicated [S3A Committers](committers.md).
 
 ## <a name="tuning"></a> Options to Tune
 
-### <a name="pooling"></a> Thread and connection pool settings.
+### <a name="flags"></a> Performance Flags: `fs.s3a.performance.flag`
+
+This option takes a comma separated list of performance flags.
+View it as the equivalent of the `-O` compiler optimization list C/C++ compilers offer.
+That is a complicated list of options which deliver speed if the person setting them
+understands the risks.
```

Comment:
nit: just in case if it looks better? `if the person or client application setting them`

Reply:
no, its an individual who takes the blame